scripts/tag_release.sh

```diff
@@ -68,7 +68,7 @@ if [[ $CONFIRM_PUSH != "" ]]; then
 fi
 
 echo "Pushing branch and tags..."
-# git push
-# git push --tags
+git push
+git push --tags
```

Comment:
Instead of this, i think it's better to just push the tag set previously, because anyone might push more tags than wanted